Automattic/jetpack
# Subscription Form block: hide custom success message on WPcom simple sites

Fixes #22193

#### Changes proposed in this Pull Request:

* Hides the Subscription Form block's custom success message (shown after submitting the form) on WordPress.com simple sites.  As mentioned in #22193 WPcom has some logic around logged-in versus logged-out users, and we don't want to cause any friction with that more managed experience.

#### Jetpack product discussion

#22193

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* On a JN or local JP dev site you can pull down the changes and open the post editor. After inserting the Subscription Form block, click on it and you can see the `Success Message Text` setting in the block panel. If you then enter `window._currentSiteType = 'simple';` (to simulate a simple site) in your browser console, click off then click back on the block you should see that `Success Message Text` setting hidden.
